source/collections/clustered-collection.txt

```diff
@@ -0,0 +1,70 @@
+.. _clustered-collection:
+
+=============================
+Create a Clustered Collection
+=============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:manual:`Clustered collections </core/clustered-collections/>` are collections 
+with a clustered index. Clustered collections store documents ordered by :manual:`clustered index </reference/method/db.createCollection/#std-label-db.createCollection.clusteredIndex>` 
+key value. You can use clustered collections when only one clustered index is 
+necessary.
+
+Procedure
+---------
+
+.. procedure::
+   :style: connected
+
+   .. step:: Click the :guilabel:`Create Collection` button.
+
+      Select a database and from the :guilabel:`Collections` screen, click the 
+      :guilabel:`Create Collection` button.
+
+      .. figure:: /images/compass/new-collection.png
+         :figwidth: 550px
+         :alt: Create New Collection
+
+   .. step:: Enter the collection name.
+
+      .. figure:: /images/compass/create-collection.png
+         :figwidth: 550px
+         :alt: Enter Collection Name
+
+   .. step:: Specify the type of collection you want to create.
+
+      From the :guilabel:`Advanced Collection Options` drop-down, select 
+      :guilabel:`Clustered Collections`.
+
+   .. step:: (Optional) Specify the :guilabel:`name` field.
+
```

Comment:
Globally, consider rewriting the first sentence in your stepped content to state your objective in completing the step. For example, something like "Name your clustered index." rather than "Specify the name field." And then you can add "From the name field, enter ..." below. I think each step can be edited slightly in this way.

Reply:
Done.